### PR TITLE
Circle CI update

### LINKED ISSUE #2294: Circle CI breaking update
I received the following email:

Hi,

You are receiving this email because you are a member of one or more paid projects building on CircleCI that does not have pipelines enabled.

We are reaching out to let you know about two upcoming changes to CircleCI:

Beginning on March 31, 2020, we will start automatically enabling pipelines for all projects. Turning on pipelines will allow your team to access new features. Watch our webinar recording or read our docs for detailed information about the transition to pipelines and the benefits that you can expect to see.
The ability to opt-out of our new user interface will be removed starting on April 15th, 2020. Our team has been hard at work building a new interface that is more responsive and makes it easier for you to accomplish your most important tasks in CircleCI. Beginning on April 15, we will start to remove the ability to switch back to our old UI.
To get ahead of these changes we recommend that you update your project settings to enable pipelines and opt in to the new UI if you haven’t already. At the end of this message, we have included links to the settings for your projects that are not yet using pipelines to make it easy to make the switch.

We’re constantly working to improve CircleCI. These changes will help us deliver the best possible experience for our customers.

Best,
The team at CircleCI

### PR DESCRIPTION
**Pull request by corradio**
Fixes #2294 

FYI @robertahunt @madsnedergaard

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,75 +1,105 @@
...
+    docker:
+      - image: circleci/node:10.13
+    steps:
+      - checkout
+      - restore_cache:
+          key: node_modules-{{ .Branch }}-{{ checksum "web/yarn.lock" }}
+      - run:
+          command: |
+            set -eux -o pipefail
+            pushd web
```
Comment (fbarl) [inline .circleci/config.yml]: Can we remove this line?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,75 +1,105 @@
...
-          flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
-          # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
-          # flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-          pylint -E parsers/*.py -d unsubscriptable-object,unsupported-assignment-operation,unpacking-non-sequence
-          python -m unittest discover parsers/test
-          npm install -g jsonlint
-          jsonlint -q config/*.json web/locales/*.json
+      - run:
+          name: Upgrade pip
+          command: sudo pip install -q --upgrade pip
```
Comment (fbarl) [inline .circleci/config.yml]: How come `sudo` is necessary? Also for `npm install` below..

File: .circleci/config.yml (hunk 1 above)
Comment (corradio) [reply .circleci/config.yml]: the next two commands require us to be in the "web" directory. 
I can also replace by a `cd` command?

File: .circleci/config.yml (hunk 2 above)
Comment (corradio) [reply .circleci/config.yml]: Circle CI requirements.. It's the only way I could make it work. `--user` didn't cut it

File: .circleci/config.yml (hunk 1 above)
Comment (fbarl) [reply .circleci/config.yml]: Ah ok.. Yeah, I think `cd` would be more standard if we're not using the dir stack, but it's a minor details :wink:

File: .circleci/config.yml (hunk 2 above)
Comment (fbarl) [reply .circleci/config.yml]: Alright, thanks for the explanation! :+1:

**Comment by fbarl** [comment]
> Maybe I should, but I never bother rewriting the commit history in the PRs as we always squash them when we merge

Alright sure, I wasn't sure what the policy was! :+1: 

p.s. I'm aware that we squash them but that still leaves the commit description somewhat dirty (see e.g. https://github.com/tmrowco/electricitymap-contrib/commit/caafc441998bac8053c519f1930c42e300bf20da), not that it's likely that anyone would normally come back and read it :stuck_out_tongue:

**Comment by fbarl** [comment]
> Could be just a CircleCI / Github integration caching issue, but maybe something actually needs to be updated in the CircleCI settings for this to look all green.

I will approve once this is resolved, maybe it would suffice to rebase and let the whole CircleCI flow have another run.

**Comment by corradio** [comment]
Should be good now!
image

**Comment by robertahunt** [comment]
@fbarl or @corradio I am getting a build error on my very simple PR #2327, I suspect it might be the changes made here. The error I am getting is: 

```
!/bin/sh -eo pipefail
ERROR IN CONFIG FILE:
[#] required key [jobs] not found

-------
Warning: This configuration was auto-generated to show you the message above.
Don't rerun this job. Rerunning will have no effect.
false
```
Any ideas?

**Comment by robertahunt** [comment]
Nvm, just had to update the branch! Sorry!

**Pull request merged**